README.md

```diff
@@ -303,12 +69,6 @@ This package is released under the [MIT License](LICENSE.md). The material in th
 
 Post an issue here on GitHub if you have any problems, and create a pull request if you make any improvements (substantial or cosmetic) to the code that you're willing to share.
 
-## FAQs
```

Comment:
Assuming this did in fact move somewhere...

Reply:
Yes